src/test/cucumber/offline_mode.feature

```diff
@@ -34,19 +38,23 @@ Feature: Offline Mode
 		And the system is bootstrapped
 		When I enter "sdk offline disable"
 		Then I see "Online mode re-enabled!"
-		When I enter "sdk install grails 2.1.0" and answer "Y"
```

Comment:
I believe that in this situation, the candidate was not previously installed, so do not have to answer.